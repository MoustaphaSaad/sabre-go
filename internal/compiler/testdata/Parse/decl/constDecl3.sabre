const (
    size int64 = 1024
)