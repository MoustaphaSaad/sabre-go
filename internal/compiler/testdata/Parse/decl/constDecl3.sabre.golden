(ConstDecl
  (ConstDeclSpec
    (IdentifierExpr IDENTIFIER(size))
    (NamedType IDENTIFIER(int64))
    =
    (LiteralExpr LITERAL_INT(1024))
  )
)